apps/b2c/pages/auth/index.tsx

```diff
@@ -0,0 +1,26 @@
+import { useSession } from 'next-auth/react'
+import { useRouter } from 'next/router'
+
+export const REDIRECT_MAP: Record<Role, string> = {
+  user: '/',
+  public: '/',
+}
+
+const Auth = () => {
+  const { push } = useRouter()
+  const { status, data } = useSession()
+
+  if (status === 'loading') return null
```

Comment:
are we planning on adding a 'loading' state here in the future? we can use the issues page to drop plans and discussions